src/main/resources/logback.xml

```diff
@@ -11,8 +11,17 @@
         </encoder>
     </appender>
 
+    <appender name="BAKING-ENDORSING-RIGHTS" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
Why a whole new appender? I thought that adding a custom `logger` and using that would've been sufficient to identify the BER messages...

Reply:
ok, changed